bindata/v3.11.0/kube-scheduler/pod.yaml

```diff
@@ -16,8 +16,7 @@ spec:
     command: ["hyperkube", "kube-scheduler"]
     args:
     - --config=/etc/kubernetes/static-pod-resources/configmaps/config/config.yaml
-    - --secure-port=0
-    - --port=10251
```

Comment:
https://github.com/openshift/cluster-monitoring-operator/blob/172beb3e0d2cbf1f79bf61ba9c3f5bea373b4125/assets/prometheus-k8s/service-monitor-kube-scheduler.yaml#L15 relies on scraping the **insecure** port `10251` in the `openshift-kube-scheduler` namespace.

By ommitting `--port`, `hyperkube kube-scheduler` will still be listening on the insecure port `10251`. Just explicitely setting it to `--port=0` will disable the insecure port.

tldr: this _shouldn't_ break monitoring and I verified with a local scheduler that, having ommitted `--port` i stil get metrics on `127.0.0.1:10251/metrics`.

Reply:
@sttts I'm looking at our config which we point here and I don't see we set port to 10257 and insecure to 0, am I blind or something?